scripts/prepare_automatic_backups.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+export PYTHONPATH=.:$PYTHONPATH
+find core/storage -not -path "core/storage/base_model/*" -name "gae_models.py" | xargs grep -E "class .+?Model" | awk '{print $2}' | cut -d "(" -f 1 | tr '\n' ' ' | xargs python scripts/prepare_automatic_backups.py
```

Comment:
Is this passing the bucket name argument from the shell script to the Python file?

Reply:
I think all you need to do is pass $1 at the end of the line (after the py file) to solve this.

Reply:
done!